components/Profile.jsx

```diff
@@ -1,79 +1,71 @@
 import { StatusBar } from "expo-status-bar";
 import { StyleSheet, Text, View, Image, Dimensions } from "react-native";
-// import Nav from "./Nav";
-import { getData } from "../utils/pullUserInfo";
+import Nav from "./Nav";
```

Comment:
I would remove the reference to ./Nav here as it isn't used anymore - I changed the component name when writing it and we can pull the navigation bar into the Profile on another ticket?